src/mojang.rs

```diff
@@ -12,6 +13,22 @@ lazy_static! {
     static ref HTTP_CLIENT: reqwest::Client = reqwest::Client::builder().build().unwrap();
 }
 
+lazy_static! {
+    static ref MOJANG_REQ_TOTAL: IntCounterVec = register_int_counter_vec!(
+        "mojang_requests_total",
+        "Total number of requests to mojang.",
+        &["request_type", "status"],
+    )
+    .unwrap();
+    static ref MOJANG_REQ_HISTOGRAM: HistogramVec = register_histogram_vec!(
+        "mojang_request_duration_seconds",
+        "The mojang request latencies in seconds.",
+        &["request_type"],
+        vec![0.003, 0.005, 0.010, 0.015, 0.025, 0.050, 0.075, 0.100, 0.150, 0.200]
```

Comment:
I think for external web requests, higher numbers with a more even distribution would make sense. I propose: 10ms, 20ms, 30ms, 40ms, 50ms, 60ms, 70ms, 80ms, 90ms, 100ms, 150ms, 200ms